### ISSUE TITLE
[BUG] Elementor Pro 2.1 RC1 - I can't style the Products widget.

### ISSUE THREAD

**Issue by dudaster** [pojome/elementor#5106]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.

## Description

I can't style the Products widget.
https://screencast-o-matic.com/watch/cFibQiFCI3

## Steps to reproduce

Just add a Posts widget and try to style it. See video!

## Isolating the problem

- [X ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ X] This bug happens with a default WordPress theme active.
- [ X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.2.15 (CentOS)
	MySQL version: 5.1.73
	PHP Version: 5.6.32
	PHP Max Input Vars: 1000
	PHP Max Post Size: 250M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.7
	Site URL: http://eletheme.1002.ro
	Home URL: http://eletheme.1002.ro
	WP Multisite: No
	Max Upload Size: 250 MB
	Memory limit: 40M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Twenty Seventeen
	Version: 1.6
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 2.1.3
		Author: Elementor.com

	Elementor Pro
		Version: 2.1.0-RC1
		Author: Elementor.com

	WooCommerce
		Version: 3.4.3
		Author: Automattic


== Debug ==
	Errors: 
		TypeError: Cannot read property 'tipsy' of undefined
			at http://eletheme.1002.ro/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.1.0-beta1 - 2:50873
			2018-06-25 16:56 +00:00

		TypeError: Cannot read property 'conditions' of undefined
			at http://eletheme.1002.ro/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.0.0-beta2 - 2:41382
			2018-03-28 11:39 +00:00
			x 18 times

		TypeError: Cannot read property 'controls' of undefined
			at http://eletheme.1002.ro/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.0.0-beta4 - 2:8815
			2018-03-23 09:43 +00:00

		Error: An "el" #elementor-panel-elements-search-area must exist in DOM
			at http://eletheme.1002.ro/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5 - 22:13466
			2018-03-23 09:24 +00:00

				
```
</details>

**Comment by dudaster** [comment]
I've installed RC3 on another server, fresh an clean wordpress no plugins, just woocommerce and the issue it is still present. Am I doing something wrong? Is there a check box that I'm missing?

**Comment by arielk** [comment]
You're right, we fix it for the next RC version of E-Pro 2.1
Thanks for your feedback!
